.env.example

```
@@ -203,3 +203,11 @@ NEXT_PRIVATE_LOGGER_FILE_PATH=
 
 # [[PLAIN SUPPORT]]
 NEXT_PRIVATE_PLAIN_API_KEY=
+
+# --- MALWARE SCANNING ---
+# Set to 'true' to enable ClamAV scanning.
+# Requires the 'clamav' service to be running in Docker.
+#tweak this with respect to your preferences
+NEXT_PRIVATE_ENABLE_MALWARE_SCAN=false
+NEXT_PRIVATE_CLAMAV_HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PRIVATE_CLAMAV_HOST key should go before the NEXT_PRIVATE_ENABLE_MALWARE_SCAN key

(UnorderedKey)